[Canvas] Expression metric (#104390)
elastic/kibana: src/plugins/expression_metric/README.md

# expressionMetric

> Expression Metric plugin adds an `metric` renderer to the expression plugin. @todo write a description.

Reviewer: @crob611, could you, please, help me with a description of this plugin? Thank you)

Reviewer 2: ```suggestion
Expression Metric plugin adds a `metric` renderer and function to the expression plugin. 
```

Reviewer 2: A simple description is fine, I suggested a minor change to yours and that should be fine
